### DISCUSSION TITLE
Azure AMQPNetLite.Core Client Integration for Aspire?

### DISCUSSION THREAD

**Discussion by jeroenjanssen-cpp** [microsoft/aspire#6779] [Q&A] [answered]
Hi,

We are using the Azure AMQPNetLite.Core library, is this library already on a TODO list somewhere for getting an Aspire Integration by any chance?

**Comment by davidfowl** [comment, marked as answer]
No, but consider the https://devblogs.microsoft.com/dotnet/introducing-the-dotnet-aspire-community-toolkit/ for your integration ideas
